recipes/cards/ivr_node/ivr_node.mdx

```diff
@@ -204,3 +204,23 @@ A node in IVR used to terminate the call.
   <p class="d-fs14">Description</p>
 </dt-recipe-ivr-node>
 ```
+
+## Node with a different top connector
+
+A node in IVR like branch that has a condition instead of dtmf connector.
+
+<Canvas>
+  <Story id="recipes-cards-ivr-node--node-with-top-connector" />
+</Canvas>
+
+```html
+<dt-recipe-ivr-node
+  :node-label="Branch"
+  :node-type="branch"
+>
+  <template #connector>
+    <div class="d-zi-base1 d-ai-center d-jc-center d-d-flex d-px8 d-h24 d-bar-pill d-bc-purple-600 d-bgc-purple-600
```

Comment:
update example to show use of ivr-connector

Reply:
oops! done